Shared/Mirror/mirrorClient.js

```diff
@@ -0,0 +1,282 @@
+//
+//  mirrorClient.js
+//
+//  Created by Patrick Manalich
+//  Edited by Rebecca Stankus on 8/30/17.
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+"use strict";
+
+(function () { // BEGIN LOCAL SCOPE
+
+    // VARIABLES
+    /* globals utils, Render */
+    var _this = this;
+    var debug = false;	// If true then a spectator camera model and box entity will be rezzed
+    var mobileSpectatorCamera = true;	// If true then the spectator camera will follow your avatar's position
+    var resolution = 1024;	// The resolution of the mirror when turned on
+    var mirrorOverlayID;	// The entity ID of the overlay that displays the mirror reflection
+    var mirrorOverlayRunning;	// True if mirror overlay is reflecting, false otherwise
+    var mirrorOverlayOffset = 0.01;	// The distance between the center of the mirror and the mirror overlay
+    var zeroRot = { w: 1, x: 0, y: 0, z: 0 };	// Constant quaternion for a rotation of 0
+    var mirrorToggleOverlayID;	// The entity ID of the mirror toggle overlay
+    var intervalID;	// The ID of the interval timer used for clearing in unload
+    var editorToClientChannel;	// The channel where mirrorScaler.js sends messages to mirrorClient.js
+    var clientToServerChannel;	// The channel where mirrorClient.js sends messages to mirrorServer.js
+    var serverToClientChannel;	// The channel where mirrorServer.js sends messages to mirrorClient.js
```

Comment:
serverToClientChannel and editorToClientChannel don't seem to be used, only registered? or am I seeing this wrong?

Reply:
@thoys probably remnants of when the rescale handles were attached. @RebeccaStankus let's remove those lines